common/include/pcl/point_cloud.h

```diff
@@ -595,6 +595,12 @@ namespace pcl
       inline void
       assign(InputIterator first, InputIterator last, index_t new_width)
       {
+        if (new_width == 0) {
+          PCL_WARN("Assignment with new_width equal to 0,"
+                   "setting width to size of the cloud and height to 1");
```

Comment:
```suggestion
                   "setting width to size of the cloud and height to 1\n");
```
Same below, otherwise looks good